locale/de/translation.json

```diff
@@ -8,7 +8,6 @@
     "Username or E-mail address:": "Benutzername oder E-Mail Adresse:",
     "Password:": "Passwort:",
     "Forgot Password?": "Passwort vergessen?",
-    "If you have not created an account yet, then please <1>register</1> first.": "Wenn du noch kein Nutzerkonto angelegt hast, <1>registriere</1> dich bitte zunächst.",
     "Edit": "Bearbeiten",
```

Comment:
@goapunk Now the Trans component isnt picked up anymore ... ah! https://github.com/liqd/zt-app/blob/d39ca90fa6ce75b06a8f66232d47127ad19c49ae/src/containers/Auth/LoginScreen.js#L81
Do you have an idea how to fix that?

Reply:
Awesome, thanks!